packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -82,12 +82,21 @@ export default class GoogleCalendarService implements Calendar {
           timeZone: calEventRaw.organizer.timeZone,
         },
         attendees: [
-          { ...calEventRaw.organizer, organizer: true, responseStatus: "accepted" },
-          ...calEventRaw.attendees.map((attendee) => ({ ...attendee, responseStatus: "accepted" })),
+          {
+            ...calEventRaw.organizer,
+            id: String(calEventRaw.organizer.id),
+            organizer: true,
+            responseStatus: "accepted",
+          },
+          ...calEventRaw.attendees.map(({ id, ...rest }) => ({
+            rest,
+            responseStatus: "accepted",
```

Comment:
We don't need id here and not all attendees will have an id if they are new to the event

Reply:
Let's add a eslint-ignore comment then